contracts/gateway.yaml

```diff
@@ -1936,8 +1937,121 @@ paths:
                 $ref: '#/components/schemas/ErrorResponse'
             application/xml:
               schema:
-                $ref: '#/components/schemas/ErrorResponse'                
-  
+                $ref: '#/components/schemas/ErrorResponse'
+  /v0.5/patients/profile/share:
+    post:
+      tags:
+        - patient
+        - hip facing
+      parameters:
+        - $ref: "#/components/parameters/authorization"
+        - $ref: "#/components/parameters/X-HIP-ID"
```

Comment:
Isn't the profile sharable even with HIU? in that case, the header can be a X-HIP-ID or X-HIU-ID

Reply:
Done, it can be shared with HIU as well